router/router.go

```diff
@@ -56,6 +56,15 @@ type tenantStats interface {
 	UpdateWorkspaceLatencyMap(destType, workspaceID string, val float64)
 }
 
+const (
+	// transformation(router or batch)
```

Comment:
[Optional] Probably move these constants to `router/utils`

Reply:
Done. resolving this comment